sirepo/pkcli/nersc_test.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+"""Nerc test suite
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdexc, pkdp, pkdlog
+from pykern import pkio
+from pykern import pkjson
+from pykern import pksubprocess
+import sirepo.sim_data
+
+
+_SEQUENTIAL_TEST_BASH_FILE = "sequential_test.sh"
+_SEQUENTIAL_TEST_JSON = "nersc_sequential.json"
+_RUN_DIR = "sirepo_run_dir"
+_SEQUENTIAL_RESULT_FILE = "nersc_test_result.json"
```

Comment:
Keep the name consisent (indeed share it in a constant). The output might not be json so call it `nersc_sequential.log`.